src/components/carousel/index.stories.mdx

```diff
@@ -2,33 +2,45 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs';
 import { action } from '@storybook/addon-actions';
 
 import Carousel from './index.tsx';
+import FullHeight from '../fullHeight';
 import Flex from '../flex';
 
-<Meta title="Components/Data display/Carousel" component={Carousel} />
+<Meta
+  title="Components/Data display/Carousel"
```

Comment:
I would have expected that as a `pattern`. It is present in neither of the figma projects can you check with the designer to add it there, thanks

Reply:
Sure I'll reach out to Enrique